bootstrap/roles/appliance-build.bootstrap/tasks/main.yml

```diff
@@ -19,31 +19,94 @@
   register: toplevel
 
 - apt:
-    name: aptitude
+    name: software-properties-common
     state: present
+    update_cache: yes
 
-- apt:
+- apt_repository:
+    repo: ppa:ansible/ansible
     update_cache: yes
 
 - apt:
-    upgrade: yes
+    upgrade: dist
 
 - apt:
-    name: "{{ item }}"
+    name:
+      - ansible
+      - aptly
+      - awscli
+      - bc
+      - coreutils
+      - equivs
+      - gdisk
+      - git
+      - java-package
+      - jq
+      - kpartx
+      - libxt6
+      - livecd-rootfs
+      - make
+      - man
+      - openjdk-8-jre-headless
+      - pigz
+      - qemu
+      - rename
+      - shellcheck
+      - vim
+      - zfsutils-linux
     state: present
+
+- modprobe:
+    name: zfs
+    state: present
+
+- shell: |
+    ( wget -nv http://artifactory.delphix.com/artifactory/gradle-distributions/gradle-5.1-bin.zip || \
+      wget -nv https://services.gradle.org/distributions/gradle-5.1-bin.zip ) && \
+    sha256sum -c <<< '7506638a380092a0406364c79d6c87d03d23017fc25a5770379d1ce23c3fcd4d  gradle-5.1-bin.zip' && \
+    mkdir /opt/gradle && \
+    unzip -d /opt/gradle gradle-5.1-bin.zip && \
+    rm gradle-5.1-bin.zip
+  args:
+    creates: /opt/gradle
+    executable: /bin/bash
+    warn: no
+
+- shell: |
+    wget -nv -O /usr/local/bin/shfmt \
+      https://github.com/mvdan/sh/releases/download/v2.4.0/shfmt_v2.4.0_linux_amd64 && \
+    chmod +x /usr/local/bin/shfmt
+  args:
+    creates: /usr/local/bin/shfmt
+    executable: /bin/bash
+    warn: no
+
+- shell: |
+    git clone https://github.com/willthames/ansible-lint /opt/ansible-lint && \
```

Comment:
We don't have to address this here, but Ubuntu currently provides an `ansible-lint` package, perhaps we should use that instead?